src/services/housing.js

```diff
@@ -174,7 +174,87 @@ const getApartmentApplication = async (applicationID) => {
  * @return {Promise.<ApplicationDetails>[]} Application details
  */
 const getAllApartmentApplications = async () => {
-  let applicationDetailsArray = await http.get(`housing/admin/apartment/applications/`);
+  // let applicationDetailsArray = await http.get(`housing/admin/apartment/applications/`);
+  let applicationDetailsArray = [
```

Comment:
It is more debug data so that I don't have to insert a bunch of data into the database tables just for a test, but yes, same idea.
The API is functional now

I intend to remove that debug data before merging this, but I figured I would leave it there until any suggested changes have been address, so that I don't have to manually add it back if I need to test other styling changes